redis/asyncio/__init__.py

```diff
@@ -15,6 +15,7 @@
     SentinelManagedSSLConnection,
 )
 from redis.asyncio.utils import from_url
+from redis.backoff import get_default_backoff
```

Comment:
Same idea. get_default_backoff to default_backoff? IMHO it's implied.

Reply:
changed